src/ansys/units/quantity.py

```diff
@@ -37,6 +38,17 @@
 except ImportError:
     _array = None
 
+try:
+    from matplotlib.units import AxisInfo, ConversionInterface, registry
+
+    _ci = ConversionInterface
+    _ai = AxisInfo
+    _registry = registry
+except ModuleNotFoundError:
+    _ci = object
+    _ai = None
+    _registry = dict()
+
```

Comment:
Wherever we have a sequence of **straightforward** assignments like these, we might preferto  write it in the more compact form `_a, _b, _c = a, b, c`

Reply:
Done. Thank you.